sources/packages/backend/libs/test-utils/src/factories/disbursement-overaward.ts

```diff
@@ -30,5 +33,7 @@ export function createFakeDisbursementOveraward(relations?: {
   disbursementOveraward.originType =
     DisbursementOverawardOriginType.ManualRecord;
   disbursementOveraward.deletedAt = null;
+  disbursementOveraward.addedBy = relations?.addedBy;
+  disbursementOveraward.addedDate = relations?.addedDate;
```

Comment:
IMO, `addedDate` does not make sense as a `relation` option.
It does not need to be provided also as a parameter. The consumer of the method can all it and adjust it after also.